lib/Sema/TypeCheckDecl.cpp

```diff
@@ -6061,6 +6061,19 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
           new (TC.Context) OverrideAttr(SourceLoc()));
     }
 
+    // If the overridden method is declared in a Swift Class Declaration,
+    // dispatch will use table dispatch. If the override is in an extension
+    // warn, since it is not added to the class vtable.
+    // FIXME: Only warn if the extension is in another module, and if
+    // it is in the same module, update the vtable.
+    if (auto baseDecl = dyn_cast<ClassDecl>(base->getDeclContext())) {
```

Comment:
Can you please put the * after the auto. This follows LLVM style:

http://llvm.org/docs/CodingStandards.html#beware-unnecessary-copies-with-auto